check-lint

```diff
@@ -1,9 +1,73 @@
 #!/bin/sh -e
 
 TOOL_CMD=pylint
-PATH=$(basename $0)/utils:$PATH
-CONFIG_PATH="$(dirname $0)/default_configs/pylintrc"
-FILES=$(git ls-files '*.py')
+BASE_DIR=$(dirname "$0")
+CONFIG_FILE="$BASE_DIR/../avocado-static-checks.conf"
 
-echo "** Running $TOOL_CMD (using config from $CONFIG_PATH)..."
-python3 -m pylint --rcfile=$CONFIG_PATH $FILES
+# Default config path
+DEFAULT_CONFIG_PATH="$BASE_DIR/default_configs/pylintrc"
+
+# Collect all Python files in the repository
+ALL_FILES=$(git ls-files '*.py')
+
+# Create a temporary file to store files checked with custom configs
+CUSTOM_CONFIG_FILES=$(mktemp)
```

Comment:
Is there a reason not using array instead of creating temp file? IMO, it is better to store this information into a variable instead of file, because it is runtime information and it won't be used anytime when this scrip finishes.

Reply:
Hi @richtja switched to use array as per your suggestion.

Reply:
Sorry, but I can't see that in your [changes](https://github.com/avocado-framework/avocado-static-checks/compare/356815ca1505aba897edb30cf1bda1ede2dfc628..a15defb1953c66cd7b221fb0b13993e8bd777141). You are still creating tempfile with `mktemp` instead of using array.